app/src/main/java/app/revenge/manager/ui/viewmodel/home/HomeViewModel.kt

```diff
@@ -42,7 +41,7 @@ class HomeViewModel(
     private val cacheDir = context.externalCacheDir ?: File(
         Environment.getExternalStorageDirectory(),
         Environment.DIRECTORY_DOWNLOADS
-    ).resolve("BunnyManager").also { it.mkdirs() }
+    ).resolve(app.revenge.manager.BuildConfig.MANAGER_NAME).also { it.mkdirs() }
```

Comment:
This should just say "manager". Make agnostic as much as possible.

Reply:
but manager of what?
there are many manager apps